Documentation/platforms/arm/stm32h7/boards/stm32h745i-disco/index.rst

```diff
@@ -1,3 +1,43 @@
 ===================
 ST STM32H745I-DISCO
 ===================
+
+Dual core support
+-----------------
+
+It is recommended to use CMake to build firmware consisting of multiple images.
```

Comment:
Is it possible to get Makefile working too, instead of CMakefile?

Reply:
This is a Documentation PR, so I think it is okay to document how things are done now. If Makefile will be able to build multi-image firmware in the future, the documentation should be updated when that happens. I prefer not to delay improvements in Documentation while waiting for Makefile support!